### ISSUE TITLE
Cannot expand ZIP 'eta-0.7.2.1-stub-output.jar' as it does not exist.

### ISSUE THREAD

**Issue by jneira** [typelead/gradle-eta#21]
Hi, downloading the eta-init project and running `.\gradlew run` i got this error:

```
Execution failed for task ':compileEta'.
> Cannot expand ZIP 'C:\Users\Javier\dev\lang\eta\eta-init\build\eta\main\eta-0.7.2.1-stub-output.jar' as it does not exist.
```

build.gradle:
```groovy
plugins {
    id 'com.typelead.eta' version '0.6.0'
    id 'application'
}

eta {
    version = '0.7.2b1'
    etlasVersion = '1.3.0.0'
}

dependencies {
  compile eta('base:4.8.2.0')
}
```
Same for 0.6.1, 0.6.2, and 0.6.3 versions of the plugin, in windows 10.

**Comment by rahulmutt** [comment]
We just recently released `0.6.5` which seems to fix it:
```
> Task :run
The 101st prime is 547
```

EDIT: The issue was similar across platforms, not Windows-specific. It seems to work after removing the `module Main where` declaration.

**Comment by rahulmutt** [comment]
Resolved with #20.
